src/autocomplete/chip.css.ts

```diff
@@ -13,7 +13,7 @@ export default css`
 		min-width: 40px;
 		padding: 0 4px 0 8px;
 		gap: 4px;
-		cursor: pointer;
```

Comment:
You dropped `cursor: pointer;` during rebase, please put it back.

Reply:
Addressed